modules/lib/src/main/resources/assets/js/app/rendering/UriHelper.ts

```diff
@@ -40,4 +40,8 @@ export class UriHelper {
         return adminUrl + (adminUrl.charAt(adminUrl.length - 1) === Path.DEFAULT_ELEMENT_DIVIDER ? '' : Path.DEFAULT_ELEMENT_DIVIDER) +
                baseUrl;
     }
+
+    public static buildWidgetUri(widgetUrl: string): string {
+        return `/admin/tool/${widgetUrl}`;
+    }
```

Comment:
This will not work when admin is mounted on root via vhost config

```
mapping.admin.host = localhost
mapping.admin.source = /
mapping.admin.target = /admin
mapping.admin.idProvider.system = default
```